### DISCUSSION TITLE
Une fonctionnalité "Publier" ?

### DISCUSSION THREAD

**Discussion by lynch4fr** [dbarzin/mercator#1008] [Ideas]
Bonjour,
Quand nous saisissons de nouvelles applications _(mais ça peut être aussi la cas pour les processus)_  serait il possible de ne pas les rendre tout de suite visible dans les vues ? Car parfois ils/elles sont en cours de rédaction par le cartographe.

Je pensais comme à un attribut "Publier : on/off " ?

Merci A+

**Comment by dbarzin** [comment]
Merci pour ta suggestion ! L’idée d’un attribut "Publier : on/off" est intéressante et pertinente. Ce ne sera pas l’intégrer à court terme, car cela implique pas mal de changements dans les autres vues pour cacher les applications masquées. Mais Mercator dispose d’une API qui permettrait de gérer ce type de besoin en attendant via une sorte de front-end de publication.

Je note ta proposition pour une future version. Merci pour ta contribution et ton implication !
